pkgs/by-name/su/substudy/package.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+
+, ffmpeg
+, makeWrapper
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "substudy";
+  version = "0.4.5";
+
+  src = fetchFromGitHub {
+    owner = "emk";
+    repo = "subtitles-rs";
+    rev = "${pname}_v${version}";
+    hash = "sha256-fx/Oaylh4XRjlgLJrICyADfyilFU+6INEEwdm+PB6WA=";
+  };
+
+  cargoHash = "sha256-AaopA555LJVLb8HFMHkYnAbQjuT02sSiHvTNG84H7wU=";
+
+  nativeBuildInputs = [
+    ffmpeg
+    makeWrapper
+  ];
+
+  postFixup = ''
+    wrapProgram "$out/bin/substudy" \
+      --prefix PATH : ${lib.makeBinPath [ ffmpeg ]}
+  '';
+
+  meta = with lib; {
```

Comment:
Is there something that could be added as `meta.mainProgram`?

Reply:
Yes, there is only one binary with the same name. Added.